packages/scripts/network-setup/deployLib.ts

```diff
@@ -53,6 +67,17 @@ export const deployTeritoriEcosystem = async (
 
   console.log("Wallet address:", walletAddr);
 
+  return { network, walletAddr };
+};
+
+export const deployTeritoriEcosystem = async (
+  opts: DeployOpts,
+  networkId: string,
+  wallet: string,
+) => {
+  const { network, walletAddr } = await initDeploy({ opts, networkId, wallet });
+  console.log("Wallet address:", walletAddr);
```

Comment:
before merge, clear console.log statements

Reply:
These ones are not used to debug, but to trace the script execution IMO

Reply:
hmm i see,  maybe we could use a logger for this to ensure a consistent formatting and differents levels of lol (info, warning, error ...) i approve the PR, we can do this later if needed i think.

Reply:
We'll see in another PR and I note the need. Let's discuss about that